### PR TITLE
Fixes #12805: support-info script don't list installed plugin

### PR DESCRIPTION
**Pull request by el-k0022**
https://issues.rudder.io/issues/12805

### CODE REVIEW

File: rudder-agent/SOURCES/debug-script/lib/collect [hunk 1]
```diff
@@ -64,6 +64,8 @@ collect_logs() {
   then
     collect_logs_annex_header "installed rudder plugins"
     ls -l /opt/rudder/share/plugins >> "${LOG_FILE}" 2>&1
```
Comment (peckpeck) [inline rudder-agent/SOURCES/debug-script/lib/collect]: Remove this line

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 2]
```diff
@@ -49,6 +49,16 @@ if [ ! -d /var/rudder/configuration-repository/techniques ]; then
   touch /opt/rudder/etc/force_technique_reload
 fi
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-webapp-postinst]: This should not be here

**Comment by el-k0022** [comment]
Commit modified

**Comment by peckpeck** [comment]
You still have a 2nd commit that should not be there

**Pull request merged**